Assets/Scripts/Player/PlayerMovement.cs

```diff
@@ -0,0 +1,27 @@
+using UnityEngine;
+
+public sealed class PlayerMovement : MonoBehaviour
+{
+    [SerializeField] private float movingSpeed;
+    
+    private CharacterController _characterController;

```

Comment:
Ik zou zoiezo deze veranderen naar een RidigBpdy2D. CharacterVontroller is voor 3D, maar we hebben een 2D game.

Reply:
CharacterController is ook voor 2D